app/Providers/AuthServiceProvider.php

```diff
@@ -12,6 +15,7 @@ class AuthServiceProvider extends ServiceProvider
 	 * @var array<string, string>
 	 */
 	protected $policies = [
+		// User::class => UserPolicy::class,
 		// 'App\Model' => 'App\Policies\ModelPolicy',
 	];
```

Comment:
Why do we need an empty array? Can that be nuked?

Reply:
No longer an empty array. :) 
Resolve?